airbnb-app/src/context/CalenderDateProvider.jsx

```diff
@@ -43,11 +28,30 @@ export function CalenderDateProvider({ children }) {
     }
   };
 
+  const getTimeFromInfo = Info => {
```

Comment:
인자는 보통 카멜케이스(camelCase)로 짓습니다! 파스칼케이스는 타입정의할때 더 쓰는거 같네요~
```suggestion
  const getTimeFromInfo = info => {
```